[2023-12-27 14:17:03,808] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting.merge_data 2023-12-27T14:16:53.720586+00:00 [queued]>
[2023-12-27 14:17:03,825] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting.merge_data 2023-12-27T14:16:53.720586+00:00 [queued]>
[2023-12-27 14:17:03,825] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 14:17:03,825] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2023-12-27 14:17:03,825] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2023-12-27 14:17:03,833] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): merge_data> on 2023-12-27T14:16:53.720586+00:00
[2023-12-27 14:17:03,838] {standard_task_runner.py:52} INFO - Started process 1079 to run task
[2023-12-27 14:17:03,841] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'meeting', 'merge_data', '2023-12-27T14:16:53.720586+00:00', '--job-id', '117', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpzp5xna3j', '--error-file', '/tmp/tmp7b32f5tv']
[2023-12-27 14:17:03,843] {standard_task_runner.py:77} INFO - Job 117: Subtask merge_data
[2023-12-27 14:17:03,879] {logging_mixin.py:109} INFO - Running <TaskInstance: meeting.merge_data 2023-12-27T14:16:53.720586+00:00 [running]> on host 6fcf1040ed70
[2023-12-27 14:17:03,928] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=meeting
AIRFLOW_CTX_TASK_ID=merge_data
AIRFLOW_CTX_EXECUTION_DATE=2023-12-27T14:16:53.720586+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-27T14:16:53.720586+00:00
[2023-12-27 14:17:03,949] {python.py:151} INFO - Done. Returned value was: None
[2023-12-27 14:17:03,959] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=meeting, task_id=merge_data, execution_date=20231227T141653, start_date=20231227T141703, end_date=20231227T141703
[2023-12-27 14:17:04,013] {local_task_job.py:151} INFO - Task exited with return code 0
[2023-12-27 14:17:04,035] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
